distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Parse_To_Table.enso

```diff
@@ -1,5 +1,7 @@
 from Standard.Base import all
 
+import Standard.Base.Data.Text.Regex.Regex
```

Comment:
Not needed as in `from Standard.Base import all`.

Reply:
Done